iso8583.js

```diff
@@ -66,6 +69,8 @@ Iso8583.prototype.decode = function(buffer, $meta) {
         while (frame) {
             var fieldPattern = this.fieldPatterns[group];
             if (!fieldPattern) {
+                frame = this.footerMatcher(frame.rest);
```

Comment:
Did you test this with:

1. footer.size = 0
2. footer.size > 0 and missing frame.rest